src/Stripe.Tests.XUnit/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,6 @@
+using Xunit;
+
+// By default, group all tests in a single collection. This ensures the tests are run sequentially,
+// which is necessary because we have to change the global configuration in order to use
+// stripe-mock.
+[assembly: CollectionBehavior(CollectionBehavior.CollectionPerAssembly)]
```

Comment:
maybe add a todo to remove that once we have completed the migration to stripe-mock